wcfsetup/install/files/lib/data/style/StyleAction.class.php

```diff
@@ -371,8 +372,9 @@ protected function updateFavicons(Style $style)
                 if (\file_exists($style->getAssetPath() . "favicon.ico")) {
                     \unlink($style->getAssetPath() . "favicon.ico");
                 }
-
-                \unlink($style->getAssetPath() . "manifest.json");
+                foreach (\glob($style->getAssetPath() . "manifest*.json") as $filename) {
```

Comment:
```suggestion
                foreach (\glob($style->getAssetPath() . "manifest-*.json") as $filename) {
```

Wildcard patterns always have the risk of matching unrelated files and the dash is part of the expected file name.